flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTablePartitions.java

```diff
@@ -18,14 +18,13 @@
  */
 package org.apache.iceberg.flink;
 
-import static org.apache.iceberg.flink.FlinkCatalogFactory.CACHE_ENABLED;
```

Comment:
why this change? is it from spotless formatting?

Reply:
Because CACHE_ENABLED of FlinkCatalogFactory was removed, and this line used string from org.apache.iceberg.CatalogProperties.
```
config.put(CatalogProperties.CACHE_ENABLED, String.valueOf(cacheEnabled));
```

Reply:
got it